javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,143 @@
+             /********************************************************/
+               console.log('\n        Smallest letter in Array\n\n')
+             /********************************************************/      
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const smalleLetter=getSmallestLetter(danishWords); // returns 'ø'
+function getSmallestLetter(words){
+    let temp=danishWords[0];
+    for(let i=0;i<danishWords.length;i++){
```

Comment:
In the first loop iteration your code will always compare the first word with itself!
E.g., here in the first iteration both word and shortestWord are "bil"
Which means there is always one unnecessary iteration. Can you change the for loop such that this is solved?

Reply:
done